scipy-1.0/scipy-optimize.tex

```diff
@@ -83,7 +85,7 @@ \subsubsection*{Nonlinear Optimization}
     iterate is sufficiently close to the solution: linear (L), superlinear (S) and quadratic (Q). Convergence rates denoted S$^*$ indicate that the algorithm
     has a superlinear rate for the parameters used in SciPy, but can  achieve a quadratic convergence rate with other parameter choices.
     \textit{Global convergence} is marked for the algorithms with guarantees of convergence to a stationary
-    point (i.e. a point $x^*$ for which $\nabla f(x^*) = 0$). The table also indicates which algorithms
+    point (i.e. a point $x^*$ for which $\nabla f(x^*) = 0$); this is \emph{not} a guarantee of convergence to a local or global minimum. The table also indicates which algorithms
```

Comment:
I think it would be better to say ``this is \emph{not} a guarantee of convergence to a global minimum.``

The convergence toward a stationary point guarantee that the solver converges towards either a local minimum or a saddle point. So saying that it is not a guarantee of convergence toward a local minimum may not be wrong but may be misleading.

Reply:
Basically because  while ``global convergence`` is not the same as convergence toward a local minimum, it is the next best thing.  If you have ``global convergence`` and the Hessian at the stationary point is positive definite than you have converged to the local minimum.

In some cases, you even have proof in the original paper of the convergence to a local minimum (and not just global convergence) and I have just said you have global convergence, for simplicity (e.g. trust-exact).

Hence when you say: ``this is \emph{not} a guarantee of convergence to a local or global minimum`` you are not wrong, but you put both statements in the same level. And while you have no ways at all of checking or guaranteeing global convergence, you can always check and, some time, even guarantee local convergence.

Reply:
I agree those two should not be put at the same level. I just didn't want to mislead people into believing there is a guarantee of convergence to a local minimum. 

But you're right, it's more important to emphasize that they aren't guaranteed a global minimum. While a local minimum isn't necessarily guaranteed either, that's not as important to emphasize. The careful reader can infer that from what you wrote about convergence to a stationary point.